server/bin/null-basket-server.js

```diff
@@ -95,12 +95,25 @@ function initApp() {
     var params = req.body;
     var email = params.email;
     var user = userData[email];
+    // Basket accepts either an explicit language choice,
+    // or an "accept_lang" preference string from which it
+    // will choose the best language.,
+    var lang = params.lang;
+    if (! lang) {
+      lang = params.accept_lang;
+      if (lang) {
+        lang = lang.split(/[\s\-;,]/)[0];
```

Comment:
>  Shouldn't we just send the first choice locale

We send the unmodified accept-language header, let basket parse it and figure out what language to use from there.

I wanted to put some logic here in the null-basket-server for testing purposes, but (IIUC) the precise logic in this part doesn't matter too much because it's only for use in our own functional tests.